plugins/nf-sqldb/build.gradle

```diff
@@ -13,11 +13,51 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+plugins {
+    id 'de.undercouch.download' version '4.1.2'
+}
+
+/**
+ * The following tasks download and confirm the MD5 checksum of the ZIP archive
+ * for Simba Athena JDBC driver and extract its contents to the build directory
+ */
+task downloadZipFile(type: Download) {
+    src 'https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC-2.0.25.1002/SimbaAthenaJDBC-2.0.25.1002.zip'
+    dest new File(buildDir, 'SimbaAthenaJDBC-2.0.25.1002.zip')
+}
+
+task verifyFile(type: Verify, dependsOn: downloadZipFile) {
+    src new File(buildDir, 'SimbaAthenaJDBC-2.0.25.1002.zip')
+    algorithm 'MD5'
+    checksum '9dfef9e578a709b863b1135a21bb3857'
+}
+
+task downloadAndUnzipFile(dependsOn: [downloadZipFile, verifyFile] , type: Copy) {
+    from zipTree(downloadZipFile.dest)
+    into "${projectDir}/libs"
+}
+
+task copyAthenaSimbaJar (dependsOn: downloadAndUnzipFile, type: Copy) {
+    from "${projectDir}/libs/SimbaAthenaJDBC-2.0.25.1002/AthenaJDBC42_2.0.25.1002.jar"
+    into "${projectDir}/libs"
+}
+
+task cleanUpLibsFolder (dependsOn: copyAthenaSimbaJar, type: Delete) {
+    delete "${projectDir}/libs/SimbaAthenaJDBC-2.0.25.1002"
+}
+
```

Comment:
Okay, I've given this a shot and though the individual tasks are working fine, downloading-unzipping-copying the JAR into the right location, it seems some Gradle configuration isn't working as expected.

I run across the following exception

```


> Task :plugins:nf-sqldb:copyPluginLibs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':plugins:nf-sqldb:copyPluginLibs'.
> Could not resolve all files for configuration ':plugins:nf-sqldb:runtimeClasspath'.
   > Could not find :AthenaJDBC42_2.0.25.1001:.
     Searched in the following locations:
       - file:/Users/eklavya/projects/code/nextflow/plugins/nf-sqldb/libs/AthenaJDBC42_2.0.25.1001.jar
     Required by:
         project :plugins:nf-sqldb

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 14s
27 actionable tasks: 6 executed, 21 up-to-date

```

My suspicion is that these tasks aren't being run at all when the Gradle build at top-level `make compile` is used 🤔

Reply:
Never mind, fixed this one.